src/main/java/de/letsdev/products/dico/scanner/backend/db/TestState.java

```diff
@@ -0,0 +1,60 @@
+package de.letsdev.products.dico.scanner.backend.db;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@Entity
+public class TestState {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private Timestamp timestamp;
+
+    private String state;
+
+    private String result;
+
+    @ManyToOne
+    private Device device;
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public Timestamp getTimestamp() {
+        return timestamp;
+    }
+
+    public void setTimestamp(Timestamp timestamp) {
+        this.timestamp = timestamp;
+    }
+
+    public String getState() {
+        return state;
+    }
+
+    public void setState(String state) {
+        this.state = state;
+    }
+
+    public String getResult() {
+        return result;
+    }
+
+    public void setResult(String result) {
+        this.result = result;
+    }
+
+    public Device getDevice() {
+        return device;
+    }
+
+    public void setDevice(Device device) {
+        this.device = device;
+    }
```

Comment:
toString equals und hashcode implementieren. kann man in der ide generieren

Reply:
done, thanks